docs/INSTALL.md

```diff
@@ -86,67 +79,73 @@ Here is the list of DC/OS versions supported on dcos-terraform natively:
 
 **Note**: Master DC/OS version is not meant for production use. It is only for CI/CD testing.
```

Comment:
<p class="message--note"><strong>NOTE: </strong>Master DC/OS version is not meant for production use. It is only for CI/CD testing.</p>

Reply:
Remove line #80 and copy-paste the above line.

Reply:
Done